zkevm/src/prover.rs

```diff
@@ -224,7 +226,14 @@ impl Prover {
         let agg_params =
             load_or_create_params(params_fpath, *AGG_DEGREE).expect("failed to init params");
         let seed = load_seed(seed_fpath).expect("failed to init rng");
-        Self::from_params_and_seed(params, agg_params, seed)
+        Self::from_params_and_seed(params, Some(agg_params), seed)
+    }
+
+    pub fn get_agg_params(&self) -> &ParamsKZG<Bn256> {
+        match &self.agg_params {
+            Some(params) => &params,
+            _ => panic!("agg params must be available"),
+        }
```

Comment:
Please remove this

Reply:
By using this function, since agg_params are released, it becomes possible to quickly discover problems that may occur if used.